utah-design-system-website/src/react/components/websiteContent/library/components/forms/DateInput/DateInputDocumentation.jsx

```diff
@@ -1,17 +1,57 @@
 /* eslint-disable max-len */
+import { CalendarInput } from '@utahdts/utah-design-system';
 import { Link } from 'react-router-dom';
+import { useImmer } from 'use-immer';
 import dateInputScreenshot from '../../../../../../../static/images/screenshots/components/dateInput/dateInput.jpg';
 import { LightBox } from '../../../../../lightbox/LightBox';
 import { pageUrls } from '../../../../../routing/pageUrls';
+import { SandboxExample } from '../../../../../sandbox/SandboxExample';
 import { StaticExample } from '../../../../../staticExamples/StaticExample';
+import { DateInputExampleCodeReact } from './DateInputExampleCodeReact';
+import { DateInputExampleProps } from './DateInputExampleProps';
+import { DateInputExampleRender } from './DateInputExampleRender';
 
 export function DateInputDocumentation() {
+  const [calendarDate, setCalendarDate] = useImmer(/** @type {string | null} */(null));
   return (
     <div className="documentation-content">
       <h1 id="h1-top">Date Input</h1>
       <p className="lead-in">The Date Input is a specialized input field that can present a date picker popup to the user.</p>
       <hr />
-      <h2 id="example">Example</h2>
+      <h2 id="section-example">Example</h2>
+
+      <div style={{ width: '200px' }}>
```

Comment:
This needs to be moved to a static example. We also need a couple more static examples. And the mockups don't match anymore 
### 😬

Reply:
3 static examples total:
- Calendar Input
- Date Input (without popup)
- Date Input with popup